code-design/README.md

```diff
@@ -1,10 +1,25 @@
-# Object-Oriented Design
+# Code Structure
+
+## Packwerk
+
+At Buoy we utilize [packwerk] to enable us to organize our code into
+packages based on domain concepts or business flows.
+
+### Guide
```

Comment:
We had mentioned an `avoid` here, wasn't it around making team based packs?

Reply:
Done